.github/workflows/docker.yaml

```diff
@@ -25,42 +102,61 @@ jobs:
         env:
           DOCKER_IMAGES: "ghcr.io/${{ github.repository }}"
         run: |
-          case $GITHUB_REF in
```

Comment:
TBH I would prefer keeping docker workflows as close to each other as possible. If there are benefits of making this change, let's discuss that and make the changes everywhere. From what I can tell, most of these changes are just reorganization of the same code.

Reply:
Yeah, this is what I mentioned that I started refactoring more heavily to support multiple registries - due to not knowing docker login supports multiple registries at once in GHA - and after that I started reverting those and thought it is okay if I leave it in this state, but then I revert everything and see what else do I have to do in other steps.

I don't think the changes were unreasonable or completely opinionated (safer tag name support, refactoring yoda condition into non-yoda condition, printing outputs for easier debugging, sorting outputs by name for easier read/write experience), but I'm not attached to these changes, and understand your preference for keeping this step as is, it's fine by me.

Fixed in  2f792d3.